File: 2022 maj/python/1.py
with open("liczby.txt", "r") as f:
    file = f.read().strip().split('\n')

cnt, ans = 0, ""
for n in file:
    if n[0] == n[-1]:
        cnt += 1
        if len(ans) == 0:
            ans = n
print(cnt, ans)

File: 2022 maj/c++/1.cpp
#include <iostream>
#include <fstream>
using namespace std;


int main()
{
    ifstream in("liczby.txt");
    
    string s, ans;
    int cnt = 0;
    while (in >> s)
    {
        if (s[0] == s[s.size() - 1])
        {
            cnt++;
            if (ans.size() == 0)
                ans = s;
        }
    }
    cout << cnt << ' ' << ans << '\n';
}